### ISSUE TITLE
Modular version of "webix_debug.js"

### ISSUE THREAD

**Issue by rognoni** [webix-hub/webix#252]
Hello, to help people to contribute, is possible to share the modular version of the big source-code file:
- common_helpers.js
- ui_helpers.js
- ajax.js
- clipbuffer.js
- color.js
- csv.js
- editors.js
- env.js
- history.js
- html.js
- i18n.js
- locale.js
- markup.js
- promise.js
- rules.js
- storage.cookie.js
- storage.local.js
- storage.session.js
- ui.accordion.js
- ui.accordionitem.js
- ui.barcode.js
- ui.button.js
- ui.calendar.js
- ui.carousel.js
- ui.chart.js
- ui.checkbox.js
- ui.colorboard.js
- ui.colorpicker.js
- ui.combo.js
- ui.context.js
- ui.contextmenu.js
- ui.counter.js
- ui.datasuggest.js
- ui.datatable.js
- ui.dataview.js
- ui.datepicker.js
- ui.fieldset.js
- ui.form.js
- ui.gridsuggest.js
- ui.grouplist.js
- ui.headerlayout.js
- ui.htmlform.js
- ui.icon.js
- ui.iframe.js
- ui.label.js
- ui.layout.js
- ui.list.js
- ui.menu.js
- ui.multicombo.js
- ui.multiselect.js
- ui.multisuggest.js
- ui.multitext.js
- ui.multiview.js
- ui.organogram.js
- ui.pager.js
- ui.popup.js
- ui.portlet.js
- ui.property.js
- ui.radio.js
- ui.rangeslider.js
- ui.resizer.js
- ui.richselect.js
- ui.scrollview.js
- ui.search.js
- ui.segmented.js
- ui.select.js
- ui.slider.js
- ui.spacer.js
- ui.submenu.js
- ui.suggest.js
- ui.tabbar.js
- ui.tabview.js
- ui.template.js
- ui.text.js
- ui.textarea.js
- ui.toggle.js
- ui.toolbar.js
- ui.tooltip.js
- ui.tree.js
- ui.treetable.js
- ui.unitlist.js
- ui.uploader.js
- ui.video.js
- ui.window.js
- Mixins.js

**Comment by social4hyq** [comment]
It's better to use es6 modules

**Comment by social4hyq** [comment]
Hi, any update for this?

**Comment by HannaBabrouskaya** [comment]
We plan to switch to es6 modules in version 5.0

**Comment by HannaBabrouskaya** [comment]
@Zwillinge @mkozhukh

**Comment by dandv** [comment]
Modularity has been [implemented in version 6](https://docs.webix.com/desktop__custom_build.html), but I see only moderate size reduction when eliminating every single components except one - `datatable`: the build goes from 676kB minified to 418kB.

**Comment by dandv** [comment]
@mkozhukh: can this issue be closed?

**Comment by Zwillinge** [comment]
yep, the issue can be closed. 

@dandv the size can be significantly reduced by removing unnecessary modules (mixins) as well.
